File: week-02/day-2/draw-pyramid.py
# Write a program that reads a number from the standard input, then draws a
# pyramid like this:
#
#
#    *
#   ***
#  *****
# *******
#
# The pyramid should have as many lines as the number was

number = 15 #int(input("enter a number: "))

x = 1
while (x <= number):
    print(("*" * x) - 1)
    x = x + 3

File: week-07/day-2/draw-pyramid.js
'use strict';

// Write a program that draws a
// pyramid like this:
//
//    *
//   ***
//  *****
// *******
//
// The pyramid should have as many lines as lineCount is

function drawPyramid (lineCount) {
    var space = ' ';
    var star = '*';
    for (var line = 1; line < lineCount + 1; line++) {
        console.log(space.repeat(lineCount - line) + star);
        star += '**';
    }
}

drawPyramid (5);
